.eslintrc.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
     '@typescript-eslint/explicit-function-return-type': 'off',
     '@typescript-eslint/explicit-module-boundary-types': 'off',
     '@typescript-eslint/no-explicit-any': 'off',
-    'prettier/prettier': 0
+    'prettier/prettier': 0,
+    semi: ["error", "always"]
```

Comment:
semi is not consistent with rest of the settings. could you wrap it in ' ?

Reply:
Pretter was disabled in previous PR. Please remove it from project dependencies as it's not used anymore.